### PR TITLE
py-transformers: add v4.35.2

### PR DESCRIPTION
**Pull request by adamjstewart**
Current version seemed to be having issues in CI so I added a newer version.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-tokenizers/package.py [hunk 1]
```diff
@@ -13,23 +13,27 @@ class PyTokenizers(PythonPackage):
...
     depends_on("openssl")
+
+    # Historical dependencies
+    depends_on("py-setuptools", when="@:0.13", type="build")
+    depends_on("py-setuptools-rust", when="@:0.13", type="build")
+
+    # A nightly or dev version of rust is required to build older versions.
+    # https://github.com/huggingface/tokenizers/issues/176
+    # https://github.com/PyO3/pyo3/issues/5
+    depends_on("rust@nightly", when="@:0.10", type="build")
```
Comment (haampie) [inline var/spack/repos/builtin/packages/py-tokenizers/package.py]: https://github.com/PyO3/pyo3/commit/7075827a038826222bbc9cb6153a1d7455396531 landed in pyo3 version 0.11

but 0.11 was already pinned in https://github.com/huggingface/tokenizers/commit/16f75d9efcbd0e2f634e9756e84f8e6d88b78686

which is `python-v0.9.0` which is separately versioned? :laughing: 

I think it's good to deprecate something to get rid of this moving target.

**Comment by alalazo** [comment]
Duplicate of #41200

File: var/spack/repos/builtin/packages/py-tokenizers/package.py (hunk 1 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-tokenizers/package.py]: Am I allowed to remove it someday if I deprecate it first? 😛

**Comment by adamjstewart** [comment]
#41200 has been closed, this PR should be merged instead.

**Review by alalazo — APPROVED** [review]

**Pull request merged**